tests/automated/1_program.py

```diff
@@ -13,13 +14,40 @@ def check_signature(p):
 
 @test_all_pandas
 @panda_connect_and_init(full_reset=False)
-def test_a_recover(p):
+def test_a_known_bootstub(p):
```

Comment:
new test should go in its own PR before this one

Reply:
also can you add a description for the test?

Reply:
added description, will split PR